lib/commons/dom/is-modal-open.js

```diff
@@ -34,26 +37,30 @@ function isModalOpen(options) {
   // in the viewport. since we aren't sure if it is or is not a modal this is
   // just our best guess of being one based on convention.
 
-  if (cache.get('isModalOpen')) {
-    return cache.get('isModalOpen');
-  }
-
-  const definiteModals = querySelectorAllFilter(
-    // TODO: es-module-_tree
-    axe._tree[0],
-    'dialog, [role=dialog], [aria-modal=true]',
-    vNode => isVisible(vNode.actualNode)
+  return cache.get(
+    'isModalOpen',
+    definiteModalsExists(definiteModalRoles) ||
+      probableModalExists(modalPercent)
```

Comment:
I feel that this change could lead to causing a subtle bug. If the order of the OR conditions were changed, the code would not work as expected since `probableModalExists` would return `undefined` and then still execute `definiteModalsExists`